game_event/tests.py

```diff
@@ -179,3 +198,86 @@ def test_post_game_event(self, client, saved_court, court_ball_game):
+        client.force_login(player.user)
+        response = client.get('/game-events/-1/')
+        assert response.status_code == 404
+
+    def test_success_loading_site_join_game_event(self, client, saved_game_event, player):
+        client.force_login(player.user)
+        game_event_id = saved_game_event.id
+        url = reverse('join_event', args=[game_event_id])
+        response = client.get(url)
+        assert response.status_code == 200
+
+    def test_fail_loading_site_join_game_event(self, client, player):
+        client.force_login(player.user)
+        response = client.get('/game-events/join-event/9999/')
+        assert response.status_code == 404
+
+    def test_success_loading_site_remove_game_event(self, client, saved_game_event, saved_game_event_player):
+        client.force_login(saved_game_event_player.player.user)
+        game_event_id = saved_game_event.id
+        url = reverse('remove_from_event', args=[game_event_id])
+        response = client.get(url)
+        assert response.status_code == 200
+
+    def test_fail_loading_site_remove_game_event_from_player(self, client, player):
+        client.force_login(player.user)
+        response = client.get('/game-events/removee-from-event/-1/')
+        assert response.status_code == 404
+
+    def test_remove_landing_site_game_event(self, client, saved_game_event, saved_game_event_player):
+        client.force_login(saved_game_event_player.player.user)
+        game_event_id = saved_game_event.id
+        url = reverse('remove_from_event', args=[game_event_id])
+        response = client.get(url)
+        assert response.status_code == 200
+        saved_game_event.delete()
+        response = client.get(url)
+        assert response.status_code == 404
+
+    def test_remove_player_from_event(self, client, saved_game_event, saved_game_event_player):
+        client.force_login(saved_game_event_player.player.user)
+        game_event_id = saved_game_event.id
+        url = reverse('remove_from_event', args=[game_event_id])
+        response = client.get(url)
+        assert response.status_code == 200
+        with pytest.raises(ObjectDoesNotExist):
+            GameEventPlayer.objects.get(game_event=saved_game_event, player=saved_game_event_player.player)
+
+    def test_remove_player_that_is_not_in_event(self, client, saved_game_event, player):
+        client.force_login(player.user)
+        game_event_id = saved_game_event.id
+        url = reverse('remove_from_event', args=[game_event_id])
+        with pytest.raises(ObjectDoesNotExist):
+            client.get(url)
+
+    def test_add_player_to_event(self, client, saved_game_event, player):
+        client.force_login(player.user)
+        game_event_id = saved_game_event.id
+        url = reverse('process_answer', args=[game_event_id])
+        response = client.post(url, {'answer': 'yes'})
+        assert response.status_code == 200
+        assert 'in_event' in response.context
```

Comment:
why do we need to check that `in_event `is in? Don't we want to check the value of `in_event`?

Reply:
yes, you defiantly right, I changed it and added of GameEventPlayer has been created.